{{cookiecutter.github_repository}}/pyproject.toml

```diff
@@ -0,0 +1,127 @@
+[tool.poetry]
+name = "{{cookiecutter.github_repository}}"
+version = "{{cookiecutter.version}}"
+description = "{{cookiecutter.project_description}}"
+authors = ["{{cookiecutter.default_from_email}}"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Django = "^3.2.8"
+django-environ = "^0.8.1"
+argon2-cffi = "^21.1.0"
+python-dotenv = "^0.19.1"
+django-cors-headers = "^3.10.0"
+pytz = "^2021.3"
+psycopg2-binary = "^2.9.1"
+Pillow = "^8.4.0"
+django-extensions = "^3.1.3"
+django-uuid-upload-path = "^1.0.0"
+django-versatileimagefield = "^2.2"
+djangorestframework = "^3.12.4"
+django-rest-swagger = "^2.2.0"
+django-log-request-id = "^1.7.0"
+{%- if cookiecutter.add_celery.lower() == "y" %}
+celery = {extras = ["redis"], version = "^5.1.2"}
+{%- endif %}
+PyJWT = "^2.3.0"
+django-mail-templated = "^2.6.5"
+pydot = "^1.4.2"
+pyparsing = "^3.0.1"
+django-storages = "^1.12.2"
+boto3 = "^1.19.3"
+django-redis = "^5.0.0"
+hiredis = "^2.0.0"
+mkdocs-material = "^7.3.4"
+markdown-include = "^0.6.0"
+mkdocs-git-revision-date-localized-plugin = "^0.10.0"
+mkdocs-mermaid2-plugin = "^0.5.2"
+uvicorn = {extras = ["standard"], version = "^0.15.0"}
+flower = "1.0.0"
+Werkzeug = "^2.0.2"
+gunicorn = "^20.1.0"
+django-phonenumber-field = "^5.2.0"
```

Comment:
django-phonenumber-field, gunicon, phonenumbers look like extra dependencies not used by django-init. Also, i think it would better to have ansible as dev dependency.

Reply:
Noted. I'll update this